plugins/metrics/message.go

```diff
@@ -48,6 +74,21 @@ var (
 	// number of messages in the database at startup
 	initialMessageTotalCountDB uint64
 
+	// total number of branches in the database at startup
+	initialBranchTotalCountDB uint64
+
+	// total number of finalized branches in the database at startup
+	initialFinalizedBranchCountDB uint64
+
+	// total number of confirmed branches in the database at startup
+	initialConfirmedBranchCountDB uint64
```

Comment:
i am not sure i understand this metric.. is this loaded from the snapshot? shouldnt the number of branches be zero on startup?

Reply:
It doesn't have to be, for example when a node is restarted with the same database. When running from a snapshot with a clean database then it will be zero.